lib/vector/diglib/frmt.c

```diff
@@ -46,7 +48,10 @@ int dig_read_frmt_ascii(FILE *dascii, struct Format_info *finfo)
             return -1;
         }
 
-        strcpy(buf1, buff);
+        len = G_strlcpy(buf1, buff, sizeof(buf1));
+        if (len >= sizeof(buf1)) {
+            G_fatal_error(_("Name <%s> is too long"), buff);
```

Comment:
It is a _line_ which is too long, not a _name_.

Compare _G_getl2_ call on line 41/43 and the code in r.patch which uses "Name ...".

Reply:
done